roles/rclone_serve_restic/README.md

```diff
@@ -0,0 +1,21 @@
+# `famedly.base.rclone_serve_restic` ansible role for multi-tenancy append-only backups to s3 using rclone
+
+## Required Options:
+
+this role supports a single s3 bucket as a backend, you can set the required options like so
```

Comment:
Sentences in README should start upper-case (also see L21) and the trailing `:` on headings (L3) seems off